staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -111,7 +111,7 @@ func createHandler(r rest.NamedCreater, scope RequestScope, admit admission.Inte
 		audit.LogRequestObject(ae, obj, scope.Resource, scope.Subresource, scope.Serializer)
 
 		userInfo, _ := request.UserFrom(ctx)
-		admissionAttributes := admission.NewAttributesRecord(obj, nil, scope.Kind, namespace, name, scope.Resource, scope.Subresource, admission.Create, userInfo)
+		admissionAttributes := admission.NewAttributesRecord(obj, nil, scope.Kind, namespace, name, scope.Resource, scope.Subresource, admission.Create, len(options.DryRun) > 0, userInfo)
```

Comment:
@apelisse made a helper for this len check.

Reply:
Yeah, it's in a PR that isn't merged yet. We could solve that by merging #66466 ;-)

Reply:
> Yeah, it's in a PR that isn't merged yet. We could solve that by merging #66466 ;-)

I swear I lgtmed that.  You noted that the storage was blocked so it was still ok.